06_metrics.ipynb

```diff
@@ -0,0 +1,2393 @@
+{
```

Comment:
<p>Can you get a cluster via <a href="https://pangeo.io/cloud.html#dask-gateway" target="_blank">dask gateway</a>. That might also contribute to some errors.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>